### PR TITLE
implemented task and tests

### PR DESCRIPTION
**Pull request by ivadimko**

### CODE REVIEW

File: src/getValueAtZero.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+/**
+ * Implement getValueAtZero function:
+ *
+ * it takes another function 'f' and it's value with 0 as argument
+ *
+ * @param {function} f
```
Comment (roman-mate) [inline src/getValueAtZero.js]: f is too short for public argument. maybe 'func'?

File: src/getValueAtZero.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ *
+ * @return {function}
+ *
+ * f = x => x + 5;
```
Comment (roman-mate) [inline src/getValueAtZero.js]: they don't know lambda functions yet. 
lets add couple more examples.

Comment (ivadimko) [reply src/getValueAtZero.js]: @roman-mate please check this commit
https://github.com/mate-academy/js_get-value-at-zero/pull/1/commits/4d07f33cbdd817a96dd0dad812b47d758cae7e73

File: src/getValueAtZero.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ * it takes another function 'initialFunction' and it's value with 0 as argument
+ *
+ * @param {function} initialFunction
+ *
+ * @return {function}
+ *
+ * const initialFunction = function(x) {
+ *     return x + 5
+ * };
+ * getValueAtZero(initialFunction) === 5;
```
Comment (mgrinko) [inline src/getValueAtZero.js]: I prefer to have examples before the params

File: src/getValueAtZero.js (hunk 1 above)
Comment (mgrinko) [reply src/getValueAtZero.js]: `callback` is event better name

File: src/getValueAtZero.js (hunk 2 above)
Comment (mgrinko) [reply src/getValueAtZero.js]: @roman-mate they know arrow functions 
as they should have already passed FreeCodeCamp

**Pull request merged**